controllers/components/debezium_connector.go

```diff
@@ -48,6 +53,30 @@ func (dc *DebeziumConnector) Delete() (err error) {
 }
 
 func (dc *DebeziumConnector) CheckDeviation() (problem, err error) {
+	// leave this commented line for testing
+	// debConPtr, err := dc.KafkaClient.GetConnector("xjoinindexpipeline.hosts.1679941693938094928")
+	debConPtr, err := dc.KafkaClient.GetConnector(dc.name)
+
+	if err != nil {
+		return nil, errors.Wrap(err, 0)
```

Comment:
this should check if the connector exists and return a problem instead of an error if it doesn't exist. Similar to the avro_schema function: https://github.com/RedHatInsights/xjoin-operator/blob/master/controllers/components/avro_schema.go#L91

Reply:
something like `if !dc.Exists() {return problem, nil}`